common/src/main/java/us/ilite/common/io/CodexNetworkTablesParser.java

```diff
@@ -98,7 +101,15 @@ public String codexToCSVLog() {
      * @return File path of a certain enumeration
      */
     public File file() {
```

Comment:
When we aren't in a live match, we will have duplicate file names.  How do we plan to handle that?